snuba/datasets/processors/search_issues_processor.py

```diff
@@ -38,20 +56,23 @@ class IssueOccurrenceData(TypedDict, total=False):
 
 class IssueEventData(TypedDict, total=False):
     # general data from event.data map
-    trace_id: Optional[str]
-    platform: str
-    environment: Optional[str]
-    release: Optional[str]
-    dist: Optional[str]
     received: float
     client_timestamp: float
     tags: Mapping[str, Any]
-    user: Mapping[str, Any]  # user, user_hash, user_id, user_name, user_email
-    ip_address: str  # user.ip_address
+    user: Mapping[str, Any]  # user_hash, user_id, user_name, user_email, ip_address
     sdk: Mapping[str, Any]  # sdk_name, sdk_version
     contexts: Mapping[str, Any]
     request: Mapping[str, Any]  # http_method, http_referer
 
+    # tag aliases
+    environment: Optional[str]  # tags[environment] -> environment
+    release: Optional[str]  # tags[sentry:release] -> release
+    dist: Optional[str]  # tags[sentry:dist] -> dist
+    # (tags[sentry:user] or user[id]) -> user
+
+    # contexts aliases
+    # contexts.trace.trace_id -> trace_id
```

Comment:
Why is this and line 71 commented out? Something for later on or left in?

Reply:
Yeah, this is reserved for later since right now we're not even sending the context data through. Will add a TODO comment to be safe.